### DISCUSSION TITLE
CCH2: Linking California to "USA"

### DISCUSSION THREAD

**Discussion by Annie-Ayers** [Symbiota/community-board#593] [Wish List]
In CCH2 it seems that "California" is a drop-down selection option for "United States" but not for "USA". Is it possible to have California as a dropdown item for both?

**Comment by themerekat** [comment]
@Annie-Ayers that's a good question that points out an issue in our current geographic thesaurus. Summarized in this to-do item: https://github.com/BioKIC/Symbiota/issues/1647
